### PR TITLE
Input changes for v2

### PR DESCRIPTION
**Pull request by zijchen**
Changing the inputs based on the proposal from #98 

Major changes:
- Retired the `server-name` input. Server will be determined from the connection string.
- Removed the `dacpac-package`, `sql-file`, and `project-file` inputs. Replaced by a singular `path` input that will determine the action based on the file extension.
- Added an `action` input that maps to sqlpackage actions. Required when not using .sql file. Currently only Publish is supported. Support for other types to be added in #55 
- `arguments` renamed to `sqlpackage-arguments`

### CODE REVIEW

**Comment by zijchen** [comment]
I won't run the `pull_request_target` workflow as it uses old yaml which won't work any more. Running `pull_request` workflow instead to use latest pr-check.yml

File: src/main.ts [hunk 1]
```diff
@@ -56,76 +56,52 @@ function getInputs(): IActionInputs {
...
-            connectionConfig: connectionConfig,
-            sqlFile: sqlFilePath,
-            actionType: ActionType.SqlAction,
-            additionalArguments: additionalArguments
-        } as ISqlActionInputs;
+    let filePath = core.getInput('path', { required: true });
+    filePath = AzureSqlActionHelper.resolveFilePath(filePath);
+
+    // Optional inputs
+    const action = core.getInput('action');
```
Comment (Benjin) [inline src/main.ts:63]: Why prompt for Action if only Publish is supported?

**Review by dzsquared — APPROVED** [review]

File: src/main.ts (hunk 1 above)
Comment (zijchen) [reply src/main.ts:66]: for supporting more actions in the future
#106 will add some more

File: src/AzureSqlActionHelper.ts [hunk 2]
```diff
@@ -88,6 +89,15 @@ export default class AzureSqlActionHelper {
         return filePath;
     }
 
+    public static getSqlpackageActionTypeFromString(action: string): SqlPackageAction {
+        switch (action.trim().toLowerCase()) {
```
[the remark this answers is not in the document]
Comment (zijchen) [reply src/AzureSqlActionHelper.ts:93]: Yeah we will support more actions later

**Comment by zijchen** [comment]
Won't run `pull_request_target` as it will fail with the input changes

**Pull request merged**